packages/dds/tree/src/util/transactionResult.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Describes the result of a transaction. Transactions may either succeed and commit, or fail and abort.
+ * @alpha
+ */
+export enum TransactionResult {
+	/**
+	 * Indicates the transaction succeeded. This value is falsy.
```

Comment:
What is the motivation for making `Commit` falsy and `Abort` truthy?

Reply:
No idea.

Reply:
I have removed this. It can't even be used within our package with our current linter rules (as of today).